docs/operations.rst

```diff
@@ -173,12 +168,19 @@ currently work:
    np.max(x)
    np.min(x, axis=(0, 2))
    x.prod()
+   x.argmax()
+   x.argmin()
+   np.mean()
+   np.var()
+   np.std()
+   x.any()
+   x.all()
```

Comment:
This is already in the API docs; we don't need to make this page a complete reference.